dolphinscheduler-meter/src/main/java/org/apache/dolphinscheduler/meter/metrics/DefaultMetricsProvider.java

```diff
@@ -74,6 +77,9 @@ public SystemMetrics getSystemMetrics() {
                 .systemMemoryUsed(totalSystemMemory - systemMemoryAvailable)
                 .systemMemoryMax(totalSystemMemory)
                 .systemMemoryUsedPercentage((double) (totalSystemMemory - systemMemoryAvailable) / totalSystemMemory)
+                .diskUsed(diskToTalBytes - diskFreeBytes)
+                .diskTotal(diskToTalBytes)
+                .diskUsedPercentage((diskToTalBytes - diskFreeBytes) / diskToTalBytes)
```

Comment:
nothing but want to make sure we store as float in stock and render as '\d %' right?

Reply:
We will render as string by 
```
<Gauge
    data={(
      JSON.parse(item.heartBeatInfo).diskUsage * 100
    ).toFixed(2)}
  />
```

It works well 

<img width="1215" alt="image" src="https://github.com/user-attachments/assets/93593ab9-746e-4df4-8441-21158cf30663" />